config/application.config.php

```diff
@@ -3,6 +3,7 @@
     // This should be an array of module namespaces used in the application.
     'modules' => [
         'ZendDeveloperTools',
+        'TestData',
```

Comment:
Adding it in `application.config.php` is not such a good idea, since this will break the application when the module is not there. So it will break in non-dev environments/